modules/applications/pages/ui-builder/how-to-share-data-across-pages.adoc

```diff
@@ -20,20 +20,33 @@ image:ui-builder/guides/configure-query-params.png[configure-query-params]
 The `navigate to` function will redirect to the chosen page. The query params will be passed between the source and the target page's URL: `your-application/target-page/edit?xx=value`
 
 
-== Using JavaScript objects
+== Store data using Store functions
+
+The `storeValue()` function stores the data in the browser’s local storage as key-value pairs that represent storage objects and can be later accessed anywhere in the application.
+
+If you want to store the text of an input widget, you can use `storeValue()` as shown below:
+[source, JS]
+----
+{{storeValue('email',input1.text)}}
+----
+
+Here, `email` is the key where the value is stored, and `input1.text` is the value in the input widget that's saved in the storage object.
+
 
 You can also create a JS object to store key-value pairs within the local storage:
 [source, JS]
 ----
 export default {
      populate_store () {
          storeValue('keyName', 'value', true);
-         return appsmith.store.name;
+         return appsmith.store.keyName;
      }
  }
 ----
 
-Once done, retrieve data using: `{{appsmith.store.keyName}}` in your queries and other widgets.
+Once done, retrieve stored data using `{{appsmith.store.keyName}}` in your queries and other widgets.
 
+Instead of writing code, you can also store data from the widget's settings, see our xref:how-to-control-visibility-of-widgets.adoc#_using_the_storevalue_function[other guide] for more details.
 
+Please note that you can use the `clearStore()` function which clears all the data that was stored in the local storage of the browser, as well as the `removeValue()` function which clears the value associated with a specified key.
```

Comment:
I'm wondering if we should reference the Appsmith documentation at the end of this section.
"For more details, see https://docs.appsmith.com/reference/appsmith-framework/widget-actions/store-value"

Reply:
You're right, I've made the changed.
However I think we do want to keep a good amount of content in our documentation. From what I've noticed, customers will never think about going to the Appsmith documentation for help (on purpose). This is a Bonitasoft product so they only want to rely on the Bonitasoft documentation

Reply:
IMHO this whole section should be just
See this https://docs.appsmith.com/reference/appsmith-framework/widget-actions/store-value[dedicated guide].